src/routes/display/+layout.svelte

```diff
@@ -1,10 +1,36 @@
 <script lang="ts">
   import QR from "$lib/assets/qr-code.svg";
+  import { pb } from "$lib/stores/authStore";
+  import { onMount } from "svelte";
+
+  let screenMessage = {};
+
+  onMount(async () => {
```

Comment:
Kunne du gjort initial fetching på server-side, slik som i #28?

Reply:
Hva er motivasjonen for å hydrere først server-side?

Reply:
> Hva er motivasjonen for å hydrere først server-side?

Det er veldig mye raskere, som du kan se på forskjellen fra /drinks før og etter den pull requesten ble merget. Når jeg kjører denne nå sier teksten "undefined" i rundt 0.5 sekunder før den oppdaterer verdien.

Reply:
Jeg tenker også det burde være en boolean value i PB som sier om meldingen skal vise eller ikke, heller enn å sjekke om det er en tom string eller ikke

Reply:
er det noe forbedring om man bare bruker load mekansimen client-side? I stedet for onMount, som faktisk venter til komponentet er i DOM?

Reply:
jeg la til en boolean isVisible

Reply:
> er det noe forbedring om man bare bruker load mekansimen client-side? I stedet for onMount, som faktisk venter til komponentet er i DOM?

Await er ikke tillat utenfor en async funksion, i tillegg til at det ikke er bra å kjøre `fetch` utenfor `onMount` (eller `load`), det gir denne erroren:
```
Avoid calling `fetch` eagerly during server side rendering — put your `fetch` calls inside `onMount` or a `load` function instead
```

Reply:
> Hva er motivasjonen for å hydrere først server-side?

Etter å ha testet litt på `/drinks`, der jeg implementerte server-side fetching tidligere, fant jeg ut at med `+page.ts`, hvis jeg henter listen i `load()`, vil jeg få samme resultatet ved at den laster raskere. Dette vil nok være best å bruke her også @IldenH 

Syntax er den samme i `+page.ts` og i `+page.server.ts`.

Reply:
> Syntax er den samme i +page.ts og i +page.server.ts.

Kan vi gjøre dette overalt?